analyzers/src/SonarAnalyzer.CSharp/Rules/MethodParameterUnused.cs

```diff
@@ -256,15 +256,23 @@ private class MethodContext
         {
             public readonly SonarSyntaxNodeReportingContext Context;
             public readonly IMethodSymbol Symbol;
+            public readonly BaseMethodDeclarationSyntax MethodDeclaration;
+            public readonly LocalFunctionStatementSyntaxWrapper LocalFunctionDeclaration;
```

Comment:
Why are these split, when the only operation on them is to merge them?

Reply:
Indeed - refactored.